server/routes/createGoal/createGoalController.ts

```diff
@@ -24,7 +25,18 @@ export default class CreateGoalController {
       } as NewGoal
     }
 
-    const view = new CreateGoalView(prisonerSummary, req.session.newGoal.createGoalForm, req.flash('errors'))
+    const currentDatePlus3Months = moment().add(3, 'months').toDate()
```

Comment:
What format do these needs to be in when passed to the UI? The UI will use the `dateFilter` Nunjucks filter to render 2 dates

1. The human readable date for the Radio button label (27 January 2022)
2. The `value` of the Radio button, used to be passed into the data (presumably 2022-01-27?)